python/cuspatial/cuspatial/core/binpreds/feature_overlaps.py

```diff
@@ -20,17 +23,18 @@ class OverlapsPredicateBase(EqualsPredicateBase):
     equals predicate for all implementations up to this point.
     For example, a Point-Point Crosses predicate is defined in terms
```

Comment:
Below it says points can't overlap points.

Reply:
That is a removed block, previously I was creating the `PointPointOverlaps` predicate class, but now I just need `ImpossiblePredicate` to return `False` for all the pairs.